source/operations/cert-manager/cert-manager-operator.rst

```diff
@@ -0,0 +1,255 @@
+
+This page describes how to manage the Operator's TLS certificates with :ref:`cert-manager <minio-certmanager>`.
+
+Prerequisites
+-------------
+
+- A `supported version of Kubernetes <https://kubernetes.io/releases/>`__. 
+- `kustomize <https://kustomize.io/>`__ installed
+- ``kubectl`` access to your ``k8s`` cluster
+- Completed the steps to :ref:`set up cert-manager <minio-setup-certmanager>`
+- The MinIO Operator must not yet be installed.
+
+
+1) Create a CA Issuer for the ``minio-operator`` namespace
+----------------------------------------------------------
+
+This guide **disables** the automatic generation of certificates in MinIO Operator and issues certificates using cert-manager instead.
+
+The ``minio-operator`` namespace must have its own certificate authority (CA), derived from the cluster's ``ClusterIssuer`` certificate created during :ref:`cert-manager setup <minio-certmanager>`.
+Create this CA certificate using cert-manager.
+
+.. important::
+
+   This CA certificate **must** exist *before* installing MinIO Operator.
+
+1. If it does not exist, create the ``minio-operator`` namespace
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl create ns minio-operator
+
+2. Request a new Certificate with ``spec.isCA: true`` specified. 
+
+   This certificate serves as the :abbr:`CA (Certificate Authority)` for the `minio-operator` namespace.
+
+   Create a file called ``operator-ca-tls-secret.yaml`` with the following contents:
+
+   .. code-block:: yaml
+      :class: copyable
+      :emphasize-lines: 7,8
+
+      # operator-ca-tls-secret.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: minio-operator-ca-certificate
+        namespace: minio-operator
+      spec:
+        isCA: true
+        commonName: operator
+        secretName: operator-ca-tls
+        duration: 70128h # 8y
+        privateKey:
+          algorithm: ECDSA
+          size: 256
+        issuerRef:
+          name: selfsigned-root
+          kind: ClusterIssuer
+          group: cert-manager.io
+
+   .. important::
+
+      The ``spec.issueRef.name`` must match the name of the ``ClusterIssuer`` created when :ref:`setting up cert-manager <minio-cert-manager-create-cluster-issuer>`.
+      If you specified a different ``ClusterIssuer`` name or are using a different ``Issuer`` from the guide, modify the ``issuerRef`` to match your environment.
+
+3. Apply the resource:
+   
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f operator-ca-tls-secret.yaml
+
+Kubernetes creates a new secret with the name ``operator-ca-tls`` in the ``minio-operator`` namespace.
+
+.. important::
+
+   Make sure to trust this certificate in any applications that need to interact with the MinIO Operator.
+
+
+2) Use the secret to create the ``Issuer``
+------------------------------------------
+
+Use the secret created above to add an ``Issuer`` resource for the ``minio-operator`` namespace.
```

Comment:
```suggestion
Use the secret you just created to add an ``Issuer`` resource for the ``minio-operator`` namespace.
```
Not sure you need "you just created", but working with the confirmation of step approach you've already got going.

Reply:
Used the secret's actual name and removed the "created above" text. I like it better. Thanks!

Reply:
Even better 🙏